### ISSUE TITLE
Consider making the Search and Clear buttons position: fixed

### ISSUE THREAD

**Issue by martindholmes** [projectEndings/staticSearch#211]
Users on projects with lots of search filters on the search page often complain that although there are Search buttons at the top and the bottom, that's still a long way to scroll when you're working in the middle of the filters. Since they're always needed, Search and Clear should perhaps always be visible, in a fixed-position box on the page.

**Comment by joeytakeda** [comment]
If we want to do a full blown redesign, we could model the appearance on the filtering mechanism on Landscapes: https://loi.uvic.ca/archive/loiCollectionCustodianCaseFiles_search.html

I personally prefer the sidebar menu for filter heavy searches, especially since it’s similar to  the way library catalogues searches tend to look.

**Comment by martindholmes** [comment]
Agreed; that's a good model (although for sites with lots of filters it'll need to be more expandable).
